### ISSUE TITLE
[Feature] filter by a particular string in the body (irrespective of the title)

### ISSUE THREAD

**Issue by lamyergeier** [FreshRSS/FreshRSS#6188]
**Is your feature request related to a problem? Please describe.**
- [Filtering articles](https://freshrss.github.io/FreshRSS/en/users/10_filter.html) shows various ways to filter out feeds. But it does not include a way to filter out feeds that contain a particular string in the body (irrespective of the feed title)

**Describe the solution you’d like**
Please add `inbody` analogous to `intitle`

**Describe alternatives you’ve considered**
A clear and concise description of any alternative solutions or features you’ve considered.

**Additional context**

**Comment by Alkarex** [comment]
Let's use `intext:` to keep compatibility with Google search operators

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/pull/7228
tests welcome

**Comment by gingerbeardman** [comment]
Could this be expanded to include media:description?

A quick test shows that it currently doesn't. 

See #5528

**Comment by Alkarex** [comment]
> Could this be expanded to include media:description?

Yes it could, with a new operator though, maybe `media:***` or similar
